ios/Plugin/PrivacyScreenPlugin.swift

```diff
@@ -36,20 +42,36 @@ public class PrivacyScreenPlugin: CAPPlugin {
         guard self.isEnabled else {
             return
         }
-        self.privacyViewController = UIViewController()
-        self.privacyViewController!.view.backgroundColor = UIColor.gray
-        self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
+
         DispatchQueue.main.async {
             self.bridge?.viewController?.present(self.privacyViewController!, animated: false, completion: nil)
         }
     }
 
     @objc func onAppDidBecomeActive() {
+        guard self.isEnabled else {
+            return
+        }
+
```

Comment:
The privacy view should always be dismissed when the app becomes active. Please revert this changes.

Reply:
it was resolved in [ceec1eb](https://github.com/capacitor-community/privacy-screen/pull/51/commits/ceec1ebc2afe3a6aaa82caf8edb7c1975728c46c)

Reply:
Sorry, my fault, i mean `onAppDetectCapturing`. I will fix this. Thank you for the great work!